pkgs/games/frozen-bubble/default.nix

```diff
@@ -16,6 +16,27 @@ perlPackages.buildPerlModule {
   propagatedBuildInputs = with perlPackages; [ AlienSDL CompressBzip2 FileShareDir FileWhich IPCSystemSimple LocaleMaketextLexicon ];
 
   perlPreHook = "export LD=$CC";
+  
+  postInstall = ''
+    mkdir -p $out/share/applications
+    cat > $out/share/applications/frozen-bubble.desktop <<EOF
+    [Desktop Entry]
+    Name=Frozen-Bubble
+    Comment=Pop out the bubbles!
+    Exec=$out/bin/frozen-bubble
+    Icon=frozen-bubble
+    Terminal=false
+    X-MultipleArgs=false
+    Type=Application
+    Categories=Game;ArcadeGame;
+    Keywords=game;bubbles;pop;
+    EOF
```

Comment:
It'd be a bit cleaner to move this to a separate file and substituteAll it over into the right directory, IMO.

Reply:
nix has a `makeDesktopItem` utility to generate desktop entries when upstream hasn't any.

Reply:
Thanks for the pointer, @teto, back then I didn't know that. I'll implement your suggestion in a force-push.